chia/daemon/server.py

```diff
@@ -183,6 +184,7 @@ async def stop(self) -> Dict[str, Any]:
         await self.exit()
         if self.websocket_server is not None:
             self.websocket_server.close()
+            await self.websocket_server.wait_closed()
```

Comment:
This makes calling the `stop` function work a little better as it will wait for the websocket to close before returning, this avoiding various asyncio errors if you call `stop()` and immediate exit the program

Reply:
Thank you.

Reply:
Apparently calling this however, break `chia stop -d all` which is expecting the stop/exit command to return the response `{"success": True}` over the same websocket that we are waiting to close here.

Reply:
Added a parameter the test code uses when it explicitely calls stop